ext/opencensus_trace.c

```diff
@@ -64,6 +64,8 @@ void opencensus_trace_rinit()
 
     OPENCENSUS_G(current_span) = NULL;
     OPENCENSUS_G(trace_id) = NULL;
+    ALLOC_HASHTABLE(OPENCENSUS_G(baggage));
+    zend_hash_init(OPENCENSUS_G(baggage), 16, NULL, NULL, 0);
```

Comment:
nit:  Define baggage length as numeric constant. Will increase readability 

`#define MAX_BAGGAGE_LENGTH 16`

Reply:
this is just initial length. hashtable will grow based on allocation